Journalator/Source/Monitor/InvestONator.lua

```
@@ -0,0 +1,81 @@
+---@class JournalatorMonitorInvestONatorMixin
+JournalatorMonitorInvestONatorMixin = CreateFromMixins(JournalatorDisplayMixin)
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'JournalatorMonitorInvestONatorMixin'

(W111)